packages/zoe/src/contracts/operaConcertTicket.js

```diff
@@ -68,116 +70,98 @@ export const makeContract = harden(zcf => {
     );
     const ticketsPayment = mint.mintPayment(ticketsAmount);
 
-    const {
-      invite: contractSelfInvite,
-      inviteHandle: contractOfferHandle,
-    } = zcf.makeInvite();
+    let tempContractOfferHandle;
```

Comment:
Is it possible to rename this so that the logic is more understandable? Is that the internalTicketSupplyHandle?

Reply:
sure, that sounds good to me

Reply:
Done